webapp-django/crashstats/symbols/models.py

```diff
@@ -9,6 +9,7 @@ class SymbolsUpload(models.Model):
     filename = models.CharField(max_length=100)
     size = models.IntegerField()
     created = models.DateTimeField(default=timezone.now)
+    content_type = models.TextField()
```

Comment:
Migration missing. You need to generate one with:
```
./manage.py schemamigration symbols --auto
```

If you do that you'll run into a problem that this is a new field but it doesn't have a default or allows nulls. 

I would recommend setting `content_type = models.TextField(null=True)`.